acceptance/bundle/debug/out.stderr.txt

```diff
@@ -78,12 +78,10 @@
 10:07:59 Debug: No script defined for postinit, skipping pid=12345 mutator=initialize mutator=seq mutator=scripts.postinit
 10:07:59 Debug: Apply pid=12345 mutator=validate
 10:07:59 Debug: GET /api/2.0/workspace/get-status?path=/Workspace/Users/[USERNAME]/.bundle/debug/default/files
-< HTTP/1.1 404 Not Found
```

Comment:
Why do these logs go away?

Reply:
They don't quite go away -- they are formatted differently, seems like a line was split previously not not anymore.

Reply:
This PR does not change how debug logs are formatted, so it's not really the focus there.